validator/modules/rl/train_example.py

```
@@ -0,0 +1,452 @@
+
+def train_mlp(
+    data_dir: str = "data",
+    output_dir: str = "runs",
+    model_name: str = "mlp_example",
+    hidden: Tuple[int, ...] = (256, 256),
+    batch_size: int = 128,
+    epochs: int = 100,
+    learning_rate: float = 1e-3,
+    weight_decay: float = 1e-5,
+    validation_split: float = 0.2,
+    device: str = None,
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)